xla/service/gpu/gpu_compiler.cc

```diff
@@ -1369,15 +1369,18 @@ absl::Status GpuCompiler::OptimizeHloPostLayoutAssignment(
   const FloatSupport f8e4m3b11fnuz_support(F8E4M3B11FNUZ, F16);
   const GpuFloatSupport f8e5m2fnuz_support(gpu_version, F8E5M2FNUZ, F16);
   const GpuFloatSupport f8e4m3fnuz_support(gpu_version, F8E4M3FNUZ, F16);
-  auto add_float_normalization = [&](HloPassPipeline& pipeline) {
+  auto add_float_normalization = [&](HloPassPipeline& pipeline,
+                                     bool fp8_support = false) {
```

Comment:
Maybe change "bool fp8_support" to "bool promote_fp8", and remove the default value, to make it more explicit.